AstSemantics.md

```diff
@@ -289,7 +291,7 @@ Other control constructs may yield values if their subexpressions yield values:
 
 * `block`: yields either the value of the last expression in the block or the result of an inner branch that targeted the label of the block
 * `loop`: yields either the value of the last expression in the loop or the result of an inner branch that targeted the end label of the loop
```

Comment:
Would the result of the loop also need to be changed above? There could no longer be 'an inner branch that targeted the end label of the loop'?

Might it be a complementary simplification to also have `loop` return the `void` type, so return no values? Would all the phi's be in local variables anyway, and perhaps loops are expected mostly end with `br_if.